tests/model_registry/rbac/test_mr_rbac.py

```diff
@@ -0,0 +1,271 @@
+import pytest
+from typing import Self
+from simple_logger.logger import get_logger
+
+from model_registry import ModelRegistry as ModelRegistryClient
+from tests.model_registry.constants import MR_NAMESPACE
+from tests.model_registry.rbac.utils import (
+    assert_positive_mr_registry,
+    get_mr_client_args,
+    verify_group_membership,
+)
+from utilities.infra import switch_user_context
+from kubernetes.dynamic import DynamicClient
+from ocp_resources.group import Group
+from ocp_resources.model_registry import ModelRegistry
+from ocp_resources.role import Role
+from ocp_resources.role_binding import RoleBinding
+from utilities.constants import DscComponents
+from mr_openapi.exceptions import ForbiddenException
+from utilities.user_utils import UserTestSession
+from timeout_sampler import TimeoutSampler
+
+LOGGER = get_logger(name=__name__)
+NEW_GROUP_NAME = "test-model-registry-group"
+
+
+@pytest.mark.parametrize(
+    "updated_dsc_component_state_scope_class",
+    [
+        pytest.param({
+            "component_patch": {
+                DscComponents.MODELREGISTRY: {
+                    "managementState": DscComponents.ManagementState.MANAGED,
+                    "registriesNamespace": MR_NAMESPACE,
+                },
+            }
+        })
+    ],
+    indirect=True,
+    scope="class",
```

Comment:
isn't the scope already class on the fixture?

Reply:
+1, not needed.